src/Components/Media_Container/ListView.js

```diff
@@ -0,0 +1,207 @@
+  img {
+    width: 15% !important;
+    cursor: pointer;
+  }
+  /* border: 1px solid; */
+  width: 10%;
+  height: 70%;
+  display: flex;
+  align-items: center;
+  justify-content: space-evenly;
+  overflow: hidden;
+`;
+const TimeSection = styled.div`
+  /* border: 1px solid; */
+  width: 5%;
+  height: 70%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+`;
+
+const Progressbar = styled.div`
+  width: 65%;
+  height: 7%;
+  border-radius: 50px;
+  background: #908e8e;
+  padding: 1px 0px;
+  margin-left: 1%;
+`;
+
+const Progress = styled.div`
+  width: 15px;
+  height: 15px;
+  background: #000;
+  border-radius: 50%;
+  position: absolute;
+  top: 50%;
+  transform: translateY(-50%);
+`;
+
+const ListView = ({ data }) => {
+  const [activeTab, setActiveTab] = useState(null);
+  const [playing, setPlaying] = useState(false);
+  const [ButtonPlay, setButtonPlay] = useState(play);
+  const audioRef = useRef(null);
+  const [minutes, setMinutes] = useState(0);
+  const [seconds, setseconds] = useState("00");
+
+  //console.log(audioRef);
+
+  useEffect(() => {
+    if (audioRef.current) {
+      const canPlay = () => {
+        const duration = audioRef.current.duration;
+        const minutes = Math.floor(duration / 60);
+        setMinutes(minutes);
+        const seconds = Math.floor(duration - minutes * 60);
+        setseconds(seconds);
+        console.log(minutes, seconds);
+      };
+      audioRef.current.addEventListener("canplay", canPlay);
+      return () => {
+        audioRef.current.removeEventListener("canplay", canPlay);
+      };
+    }
+  }, [audioRef, minutes, seconds]);
+
+  const handleTab = (index) => {
+    setActiveTab(index);
+  };
+
+  const handlePlayButton = (activeTab) => {
+    console.log(activeTab);
+    if (playing) {
+      setPlaying(false);
+      setButtonPlay(play);
+    } else {
+      setPlaying(true);
+      setButtonPlay(pause);
+    }
+  };
+  return (
+    <React.Fragment>
+      <StyledWrapper>
+        {data.map((item, index) => {
+          return (
+            <React.Fragment>
+              <WrapperItems onClick={() => handleTab(index)}>
+                <ImgBlock>
+                  <PlayBtn>
+                    <img src={playBtn} />
+                  </PlayBtn>
+                  <img src={item.imgURL} />
+                </ImgBlock>
+                <ContentBlock>
+                  <b>Title : </b>
+                  <div>{item.title}</div>
+                </ContentBlock>
+              </WrapperItems>
+            </React.Fragment>
+          );
+        })}
+      </StyledWrapper>
+      <PlaySection>
+        <ControlSection>
+          <img
+            src={prev}
+            onClick={() => {
+              console.log(activeTab);
+              return !data[activeTab].id ? null : setActiveTab(activeTab - 1);
+            }}
+          />
+          <img src={ButtonPlay} onClick={() => handlePlayButton(activeTab)} />
+          <img
+            src={next}
```

Comment:
this should not image instead should be button containing image then add onClick event